tldw_Server_API/app/api/v1/endpoints/text2sql.py

```
@@ -48,7 +47,7 @@ async def generate(self, *, query: str, target_id: str) -> dict[str, str]:
         return {"sql": text}
 
 
-def _resolve_internal_target(target_id: str, media_db: MediaDatabase) -> tuple[str, str]:
+def _resolve_internal_target(target_id: str, media_db: Any) -> tuple[str, str]:
```

Ruff (0.15.6)

[warning] Dynamically typed expressions (typing.Any) are disallowed in `media_db`

(ANN401)